e2e/rbd.go

```diff
@@ -817,6 +817,13 @@ var _ = Describe("RBD", func() {
 				if err != nil {
 					framework.Failf("failed to delete application: %v", err)
 				}
+
+				// wait for the associated PVC to be deleted
+				err = waitForPVCToBeDeleted(f.ClientSet, app.Namespace, "mypvc", deployTimeout)
```

Comment:
lets not pass the hardcoded pvc name, read it from the app pod

Reply:
Thanks @Madhu-1, done. PTAL